### PR TITLE
Feature/tracking

### PR DESCRIPTION
**Pull request by benjamingregory**
Adds Identify calls in OAUTH and email/password sign ups(ins)

### CODE REVIEW

File: src/modules/auth/OauthSuccess/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types'
 import Token from './Token'
 import { SetData, SetUI, Track, Redirect } from 'instruments'
 import { getParams } from 'helpers/url'
+import { Identify } from '../../../instruments/analytics/track';
```
Comment (schnie) [inline src/modules/auth/OauthSuccess/index.js]: @benjamingregory you should be able to append `Identify` to the import on line 5

File: src/modules/auth/OauthSuccess/index.js [hunk 2]
```diff
@@ -42,6 +43,8 @@ class OauthSuccess extends React.Component {
     if (this.success) setUI.snackbar(this.success)
     // track event
     if (this.track) Track(this.track)
+
+    Identify(params.extras.user_id, { email: params.extras.email })
```
Comment (schnie) [inline src/modules/auth/OauthSuccess/index.js]: @benjamingregory not sure if it matters here, but should we call `Identify` (line 47) before the `Track` (line 45)?

File: src/modules/auth/OauthSuccess/index.js (hunk 1 above)
Comment (benjamingregory) [reply src/modules/auth/OauthSuccess/index.js]: Fixed

File: src/modules/auth/OauthSuccess/index.js (hunk 2 above)
Comment (benjamingregory) [reply src/modules/auth/OauthSuccess/index.js]: Fixed

**Comment by petedejoy** [comment]
I'm good with this as long as @schnie gives the thumbs up!

**Pull request merged**